libraries/AC_AutoTune/AC_AutoTune.cpp

```diff
@@ -391,14 +389,20 @@ void AC_AutoTune::control_attitude()
         test_run(axis, direction_sign);
 
         // Check for failure causing reverse response
-        if (lean_angle <= -AUTOTUNE_TARGET_MIN_ANGLE_RLLPIT_CD) {
+        if (lean_angle <= -attitude_control->lean_angle_max_cd() * AUTOTUNE_TARGET_MIN_ANGLE_RLLPIT_SCALE) {
```

Comment:
@lthall you should consider moving these protections to a virtual function so that multi and heli can implement their own protections. At a minimum, all of these new defines should be have multi and heli versions.

Reply:
At the moment these are all very close to the original value for the default lean angle max of 45 degrees.  I hoped this would be close enough to no change to not impact Heli.

I am not sure how much value there is in me making a copy of hash defines for heli that are identical to Multi. Would you be willing to add a commit or follow up PR to address any Heli interface issues?

Reply:
The problem is that 45 deg is no longer the angle max default. So this could have a big impact.